src/main/java/bot/TobyBot.java

```diff
@@ -0,0 +1,208 @@
+package bot;
+
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class TobyBot {
+    private static final String LINE = "____________________________________________________________";
+    private static final String FILE_PATH = "./data/TobyBot.txt";
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+
+    public static void main(String[] args) {
+        loadTasks();
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println(LINE);
+        System.out.println("Hello! I'm TobyBot");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
```

Comment:
Maybe this can be added to a method such as printWelcomeMessage() to improve readability.